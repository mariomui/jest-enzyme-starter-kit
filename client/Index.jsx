import React from 'react';
// import ReactDOM from 'react-dom';

const Index = () => (
  <div>
    helloddfind me
  </div>
);


// ReactDOM.render(<App />, document.getElementById('app'));

export default Index;
