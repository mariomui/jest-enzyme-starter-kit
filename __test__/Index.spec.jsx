import React from 'react';
// import ReactDOM from 'react-dom';
import Enzyme, { shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import toJson from 'enzyme-to-json'; // added this line
import Index from '../client/Index';


Enzyme.configure({ adapter: new Adapter() });


test('hey', () => {
  const tree = shallow(<Index />);
  expect(toJson(tree)).toMatchSnapshot();
});
