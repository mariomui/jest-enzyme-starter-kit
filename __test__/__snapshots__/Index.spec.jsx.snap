// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`hey 1`] = `
<div>
  noway
</div>
`;
